azure/Guidelines.md

```diff
@@ -0,0 +1,310 @@
+
+### Step 2: Document the decision
+
+As decision is being made, it should be documented.  The documentation should answer such questions as:
+
+* What versions of which APIs are being deprecated?
+* What versions of which APIs replace those being deprecated?
+* Who is impacted by deprecating these APIs?  Which internal partners?  What customers?
+* Why are the APIs being deprecated?  Support cost?  COGS?  Security or other reasons?
+* When exactly will the APIs be deprecated?
+
+### Step 3: Communicate with major partners and customers
+
+Many services will have particularly important customers or partners dependent on the versions of the APIs being deprecated. The team should make an effort to contact these early in the process and get their feedback. Ideally the feedback from these contacts can still be used to reevaluate the decision to deprecate an API or evaluate the appropriateness of the replacement APIs.
+
+### Step 4: Communicate to the Azure REST API Oversight and Azure SDK stakeholders
+
+The Azure CEC REST API review board, Azure SDK and documentation teams should be informed via e-mail of the decision. There should be a short window of time (2 work days) for these teams to provide feedback. Silence should be interpreted as assent to move to the next step. These teams may suggest coordinating multiple deprecating APIs from multiple teams in order to streamline the release and communication of changes.
+
+### Step 5: Communicate widely internally
+
+The decision should be communicated widely within Azure.  Mail describing the decision should be sent to the Azure API review board, WW – Communities and other appropriate aliases. There should be another short (2-5 working days) feedback period before moving on to the next step.
+
+### Step 6: Communicate externally, starting the 12 month clock
+
+The decision next **MUST** be communicated to customers and partners. The Service Notices site gives instructions for communicating this and other service changes. The Playbook contains a link to a submission form to request e-mail be sent.
+
+https://microsoft.sharepoint.com/teams/azurecomms/SitePages/Email-Request.aspx
```

Comment:
Let's change this to "contact the Azure API review board" on how to start communicate externally for now.

Reply:
Is that azureapirbcore or some other review board?

Reply:
Lets just say Azure API review board and make it harder for spammers to find the alias...